common/src/main/java/de/michelinside/glucodatahandler/common/receiver/NotificationReceiver.kt

```diff
@@ -0,0 +1,160 @@
+package de.michelinside.glucodatahandler.common.receiver
+
+import android.os.Bundle
+import android.service.notification.NotificationListenerService
+import android.service.notification.StatusBarNotification
+import android.util.Log
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RemoteViews
+import android.widget.TextView
+import de.michelinside.glucodatahandler.common.ReceiveData
+import de.michelinside.glucodatahandler.common.SourceState
+import de.michelinside.glucodatahandler.common.SourceStateData
+import de.michelinside.glucodatahandler.common.notifier.DataSource
+import de.michelinside.glucodatahandler.common.utils.GlucoDataUtils
+
+class NotificationReceiver : NotificationListenerService(), NamedReceiver {
+    private val LOG_ID = "GDH.NotificationReceiver"
+
+    override fun getName(): String {
+        return LOG_ID
+    }
+
+    override fun onNotificationPosted(statusBarNotification: StatusBarNotification?) {
+        if (isRegistered()) {
+            statusBarNotification?.let { sbn ->
+                if (sbn.packageName == "com.dexcom.g7") {
```

Comment:
As this is not only related to G7, I think it makes more sense to create a settings, selecting the package for parsing notification and compare name here.
But this is something, I can do, too.

Reply:
Learning android dev is fun! I can try to figure this out in a few hours!

Reply:
As you like!
I also started learning with this app, so maybe there are a lot of things to improve or which can be done in a better way...